tools/projmgr/test/data/TestSolution/PackLocking/pack_version_not_available.csolution.yml

```
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Open-CMSIS-Pack/devtools/main/tools/projmgr/schemas/csolution.schema.json
+
+solution:
+  target-types:
+    - type: CM0
+      device: RteTest_ARMCM0@>=0.1.1
+  packs:
+    - pack: ARM::RteTest_DFP
```

Is the version in the wrong yml element?
```suggestion
      device: RteTest_ARMCM0
  packs:
    - pack: ARM::RteTest_DFP@>=0.1.1
```